docs/make.jl

```diff
@@ -0,0 +1,61 @@
+using Documenter
+
+# Get TrixiBase.jl root directory
+trixibase_root_dir = dirname(@__DIR__)
+
+# Fix for https://github.com/trixi-framework/Trixi.jl/issues/668
+if (get(ENV, "CI", nothing) != "true") && (get(ENV, "TRIXIBASE_DOC_DEFAULT_ENVIRONMENT", nothing) != "true")
+    push!(LOAD_PATH, trixibase_root_dir)
+end
+
+using TrixiBase
+
+# Define module-wide setups such that the respective modules are available in doctests
+DocMeta.setdocmeta!(TrixiBase, :DocTestSetup, :(using TrixiBase); recursive=true)
+
+# Copy files to not need to synchronize them manually
+function copy_file(filename, replaces...)
+    content = read(joinpath(trixibase_root_dir, filename), String)
+    content = replace(content, replaces...)
+
+    header = """
+    ```@meta
+    EditURL = "https://github.com/trixi-framework/TrixiBase.jl/blob/main/$filename"
+    ```
+    """
+    content = header * content
+
+    write(joinpath(@__DIR__, "src", lowercase(filename)), content)
```

Comment:
I wrote this for TrixiParticles. Makes things a bit easier when multiple files are to be copied.